provisioning/windows-provision.ps1

```diff
@@ -65,6 +65,17 @@ Function AddToPathEnv($path) {
     Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newPath | Out-Null
 }
 
+# function Choco-Install {
+# 	[CmdletBinding()]
+# 	param(
+# 		[Parameter(Mandatory, ValueFromPipeline)]
+# 		[string]$PackageName
+# 	)
+#     Process {
+#         Invoke-Command & "choco.exe" install $PackageName --yes --no-progress --limit-output --fail-on-error-output
+#     }
+# }
+

```

Comment:
```suggestion
```

Reply:
I intent to add later a proper Choco-Install function, which would also look for potential CHOCOLATEY_$PackageNam.toUpper()_VERSION env var to pass as arg if exist

Reply:
(the build should pass, don't hesitate to commit this suggestion after reviewing so it can be merged right away)